.luacheckrc

```diff
@@ -2,6 +2,7 @@
 
 -- Ignore W211 (unused variable) with preload files.
 files["**/preload.lua"] = {ignore = { "211" }}
+files["src/nvim/lua/vim.lua"] = {ignore = { "122/vim" }}
```

Comment:
comment would be helpful

```suggestion
-- Allow vim module to modify itself. 
files["src/nvim/lua/vim.lua"] = {ignore = { "122/vim" }}
```

(NB: contrary to the `read_globals` config below)

Reply:
Actually, why not do this in the `vim.lua` module itself? That makes more sense.